tools/coco_visualize.py

```diff
@@ -25,14 +25,22 @@
 import numpy as np
 import pycocotools.mask as mask_utils
 from tqdm import tqdm
+from sty import bg
+import seaborn as sns
+
+
+def colors(n):
+    palette = sns.color_palette(None, n)
```

Comment:
Is it worth adding an extra dependency?

Reply:
Why not?

Reply:
Doesn't sound like an answer.

Reply:
Reason of your question is also not clear.

Reply:
replaced by random generation in hsv space